storage/plugins/archives/ArchivesPlugin.php

```
@@ -2399,25 +2463,40 @@ private function fetchArchivalUnitsForAuthority(int $authorityId): array
-            'SELECT ar.id, ar.type, ar.authorised_form, ar.dates_of_existence, aua.role
+            'SELECT ar.id, ar.type, ar.authorised_form, ar.dates_of_existence,
+                    ar.parallel_forms, ar.other_forms, ar.history, ar.functions,
+                    ar.places, aua.role
                FROM archival_unit_authority aua
                JOIN authority_records ar ON ar.id = aua.authority_id AND ar.deleted_at IS NULL
               WHERE aua.archival_unit_id = ?
               ORDER BY ar.authorised_form'
         );
         if ($stmt === false) {
-            return $rows;
+            throw new \RuntimeException(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '2477', column '23'). (undefined)

(MissingImport)